src/ExNavigationBar.js

```diff
@@ -377,7 +378,7 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     borderBottomColor: ExNavigationBar.DEFAULT_BORDER_BOTTOM_COLOR,
     borderBottomWidth: ExNavigationBar.DEFAULT_BORDER_BOTTOM_WIDTH,
-    elevation: 2,
+    elevation: 4,
```

Comment:
yeah. with white, it's not that good. Idk. I usually don't find apps with a white background on android tho.